pixl_export/src/pixl_export/main.py

```diff
@@ -0,0 +1,110 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""pixl_export module is an EHR extraction service app"""
+
+from __future__ import annotations
+
+import importlib.metadata
+import logging
+from datetime import (
+    datetime,  # noqa: TCH003, always import datetime otherwise pydantic throws error
+)
+from pathlib import Path
+
+from core.exports import ParquetExport
+from core.project_config import load_project_config
+from core.rest_api.router import router
+from core.uploader import get_uploader
+from fastapi import FastAPI, HTTPException
+from fastapi.responses import JSONResponse
+from pydantic import BaseModel
+
+from ._orthanc import get_study_zip_archive, get_tags_by_study
+
+app = FastAPI(
+    title="export-api",
+    description="Export service",
+    version=importlib.metadata.version("pixl_export"),
+    default_response_class=JSONResponse,
+)
+app.include_router(router)
+
+logger = logging.getLogger("uvicorn")
```

Comment:
May have mentioned elsewhere but worth switching over fully to using loguru for export